terminal.go

```diff
@@ -51,6 +51,11 @@ type Terminal struct {
 	// may be empty if the terminal doesn't support them.
 	Escape *EscapeCodes
 
+	// VisualLength can be override to provide a custom implementation
```

Comment:
```diff
- override
+ overridden
```

Reply:
thx! fixed! unfortunately it's not enough to fix for instance backspace